applications/openshift/general/oauth_logout_url_set/tests/ocp4/e2e.yml

```diff
@@ -0,0 +1,3 @@
+---
+default_result: FAIL
+result_after_remediation: PASS
```

Comment:
Could you include an empty line at the end, please?

Reply:
done, thank you